docs/customizing.md

```diff
@@ -69,6 +69,25 @@ of parsed commits.
 To add a new changelog type, create a new class that implements the
 [`ChangelogNotes` interface](/src/changelog-notes.ts).
 
+### Customizing Changelog Types and Releaseable Units
+
+To change the "releasable units" for your code, you can [customise the changelog sections](https://github.com/google-github-actions/release-please-action#overriding-the-changelog-sections) for your project.
+
+For example, if I wanted to release when I have `docs` changes on my Node.js project, I could update the GitHub Actions Workflow like so:
```

Comment:
```suggestion
For example, to include `docs` changes, configure the `changelog-types` configuration in the GitHub Actions Workflow like so:
```

We'd like to avoid using first-person in the guides.